### PR TITLE
Update github-style-guide.md

### PR DESCRIPTION
**Pull request by kprenger**

### CODE REVIEW

**Comment by mattylase** [comment]
!star

**Comment by BrandonWilliamsCS** [comment]
Can/should we list what things _not_ to look for (or block for)?
On a related note, I more or less gloss over a lot of the storyboard and xib files because I don't really know what to look for beyond "well, that's definitely wrong" values. Is there any we can address special file types like that?

**Comment by kprenger** [comment]
@BrandonWilliamsCS If you have examples of things that we shouldn't look for, please let me know. Currently, I have a note on the "style" item that mentions it is usually not something you block for.

I also currently have an item specific to iOS on Storyboards (which would include Xib's), but if you have other ideas, I'm open to them.

File: github-style-guide.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
 # Github style guide
 
-As Github users, we want to ensure our commit messages and PR requests follow similar guidelines across all work.
+As Github users, we want to ensure our commit messages and PR requests follow similar guidelines across all work. For process specifics, refer to our Confluence doc.
```
Comment (kprenger) [inline github-style-guide.md]: Since this is a public repo, I don't want to put direct links to our Confluence site.

**Comment by BrandonWilliamsCS** [comment]
!star

**Pull request merged**